From bcc6517853c09f979951ab483bd6560d45bf8e3f Mon Sep 17 00:00:00 2001
From: Jim Meyering <meyering@redhat.com>
Date: Thu, 20 Sep 2012 20:22:13 +0200
Subject: [PATCH 26/89] tests: avoid syntax-check failure for reversed compare
 args

* tests/t0203-gpt-create-on-min-sized-device.sh: Reverse args,
so that any diff output (upon failed test) looks sensible.
---
 tests/t0203-gpt-create-on-min-sized-device.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/t0203-gpt-create-on-min-sized-device.sh b/tests/t0203-gpt-create-on-min-sized-device.sh
index 113e191..4cec64c 100644
--- a/tests/t0203-gpt-create-on-min-sized-device.sh
+++ b/tests/t0203-gpt-create-on-min-sized-device.sh
@@ -42,6 +42,6 @@ echo Error: device is so small it cannot even accommodate GPT headers \
 # Try to create a GPT partition table in too little space.  This must fail.
 parted -s $dev mklabel gpt > out 2>&1 && fail=1
 # There must be a diagnostic.
-compare out exp || fail=1
+compare exp out || fail=1
 
 Exit $fail
-- 
1.8.5.3

